src/features/todo/list.tsx

```diff
@@ -18,9 +18,17 @@ export const TodosList = ({
     if (loading) return <Loader />;
     if (todos.length === 0) return <Subtitle>{emptyMessage}</Subtitle>;
 
+    const ordered = [...todos].sort((a, b) =>
```

Comment:
This is a helpful logic and we can not involve the client in it by moving this to the backend. An option can be added to getTodo api to order based on the priority or not.

Reply:
Adding an option for ordering todos in the backend sounds great. I think we can just move the code to the backend. I'll do it in this issue.